### ISSUE TITLE
Interactive elements hidden by the VisuallyHidden component don't receive Talkback virtual clicks on certain devices

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#3047]
# 🐛 Bug Report 

Elements like a Menu's DismissButton and the ListView's drag handle don't receive a click event when focused and triggered by Android Talkback. This seems to be device specific since it works on my Samsung Galaxy S9+ but not on other Android devices that have the same Talkback and Chrome version (e.g. Android 7.1.1 on HTC Nexus 9 using Talkback 12.1 with Chrome 100.0.4896.88). The root cause seems to be due to `clip-path` and/or the size of the element itself.

Base reproduction with a possible fix: https://codepen.io/majornista/pen/Exorwzq?editors=1111
bug filed: https://issuetracker.google.com/issues/229646660

## 🤔 Expected Behavior

Talkback should be able to trigger a click on a visually hidden element

## 😯 Current Behavior

Talkback isn't able to trigger a click on a visually hidden element on certain devices

## 💁 Possible Solution

Increasing the height/width of the visually hidden element seems to help, see code pen 

## 🧢 Your Company/Team

RSP
